react/features/base/testing/middleware.js

```diff
@@ -15,13 +18,19 @@ import logger from './logger';
  * @private
  */
 MiddlewareRegistry.register(store => next => action => {
+    let result;
+
     switch (action.type) {
     case CONFERENCE_WILL_JOIN:
         _bindConferenceConnectionListener(action.conference, store);
         break;
+    case SET_CONFIG:
+        result = next(action);
+        _bindTortureHelpers(store);
+        break;
     }
 
-    return next(action);
+    return result || next(action);
```

Comment:
This looks simpler to me! WDYT?

```
  MiddlewareRegistry.register(store => next => action => {
    switch (action.type) {
    case CONFERENCE_WILL_JOIN:
        _bindConferenceConnectionListener(action.conference, store);
        break;
    case SET_CONFIG:
        const result = next(action);

        _bindTortureHelpers(store);
        
        return result;
    }

    return next(action);
```

Reply:
That does not make lint happy, no const in case clause, that's the reason I did it this way.

Reply:
`error  Unexpected lexical declaration in case block  no-case-declarations`

Reply:
🤦

Reply:
The error was totally different, it is fixed now.